content/Editors/Code/options.md

```diff
@@ -717,3 +739,22 @@ A few things to note about this example:
   * a fully-qualified scope (i.e. `<scope>.<modifier>`, for example `member.mutable`)
   * just a scope (i.e. `member`)
   * a wildcard modifier (i.e. `*.mutable` to apply to all mutables, regardless of scope)
+
+
+### Recommended settings for larger solution/projects
+
+Here are settings recommended for larger projects to help with performance.
+
+```json
+{
+    "FSharp.codeLenses.references.enabled": false,
+    "FSharp.enableMSBuildProjectGraph": true,
+    "FSharp.enableAdaptiveLspServer": true,
+    "FSharp.fsac.conserveMemory": true
+}
```

Comment:
These are settings I use for larger projects to remain performant. Can discuss this further if necessary.